### PR TITLE
Explain how to add commands to kits

### PR DESCRIPTION
**Pull request by JasonHorkles**
First time I've created a pull request so hopefully I did everything right

### CODE REVIEW

**Review by mdcfe — APPROVED** [review]

**Review by pop4959 — DISMISSED** [review]
This is a nitpick but maybe change /ebc to /broadcast or /ebroadcast to be more clear as to what the example is executing.

And perhaps instead (or in addition to) include /give in the example since that seems to be the most common use case among server owners for commands in kits.

This isn't blocking this PR though, as it is quite minor.

**Comment by mdcfe** [comment]
> include /give in the example

I'm not sure this is the best idea, since it may confuse users as to what the "recommended" way to add items is. ItemMeta isn't perfect but we don't want to push people away from it unless what they're trying to do isn't possible.

**Comment by pop4959** [comment]
fair enough

File: Essentials/src/kits.yml [hunk 1]
```diff
@@ -23,6 +24,7 @@ kits:
       - dshovel 1 digspeed:3 name:Dwarf lore:Diggy|Diggy|Hole
       - lhelm 1 color:255,255,255 name:Top_Hat lore:Good_day,_Good_day
       - daxe:780 1
+      - /ebc {PLAYER} just got some fancy tools!
```
Comment (mdcfe) [inline Essentials/src/kits.yml]:
```suggestion
      - /broadcast {PLAYER} just got some fancy tools!
```

**Review by mdcfe — CHANGES_REQUESTED** [review]

**Comment by JasonHorkles** [comment]
Do I actually have to change anything or is it good with what you did? No idea what I'm doing 😂

**Comment by mdcfe** [comment]
You need to commit the suggestion for it to become part of the PR (I'd do it but I don't have access to a PC right now).

**Review by pop4959 — APPROVED** [review]

**Review by mdcfe — APPROVED** [review]

**Pull request merged**